lib/components/viewers/pattern-row.tsx

```diff
@@ -59,11 +60,11 @@ const renderDay = (homeTimezone: string, day: number): JSX.Element => {
  * Represents a single pattern row for displaying arrival times in the stop
  * viewer.
  */
+// eslint-disable-next-line complexity
```

Comment:
I think this can be dropped now

Reply:
Are you sure

Reply:
<img width="716" alt="image" src="https://github.com/user-attachments/assets/84854a58-ca39-458a-979b-34024974b3c5" />